script/core.hbs

```diff
@@ -86,3 +94,5 @@
   locale=(findFirst params.sdkLocaleOverride global_config.locale 'en')
   sdkVersion=global_config.sdkVersion
 }}
+
+{{> script/partials/page-specific-scripts pageTemplate=pageTemplate}}
```

Comment:
For 1 and 2, the locator bundle needs to execute after the SDK loads since the locator components are answers components. That's not necessary for the HitchhikerJS bundle because the cards are are put into a BaseCard object which isn't executed until the Answers onReady.

I like idea 3. I'll try that out. I think it might work well with the runtime config stuff too.

Reply:
Updated! DOMContentLoaded work as well and is much simpler!